autorest/models/name_converter.py

```diff
@@ -0,0 +1,186 @@
+            NameConverter._convert_language_default_name(global_parameter)
+
+
+    def _convert_operation_groups(self, operation_groups):
+        for operation_group in operation_groups:
+            if self._seen_id(id(operation_group)):
+                continue
+            NameConverter._convert_language_default_name(operation_group, pad_string="Model")
+            operation_group['language']['python']['className'] = NameConverter._to_camel_case(operation_group['language']['python']['name'])
+            for operation in operation_group['operations']:
+                if self._seen_id(id(operation)):
+                    continue
+                NameConverter._convert_language_default_name(operation, pad_string='Method')
+                for exception in operation_group.get('exceptions', []):
+                    if self._seen_id(id(exception)):
+                        continue
+                NameConverter._convert_language_default_name(operation['request'])
+                for parameter in operation['request'].get('parameters', []):
+                    if self._seen_id(id(parameter)):
+                        continue
+                    NameConverter._convert_language_default_name(parameter, pad_string="Parameter")
+                for response in operation.get('responses', []):
+                    if self._seen_id(id(response)):
+                        continue
+                    NameConverter._convert_language_default_name(response)
+
+    def _convert_schemas(self, schemas):
+        for obj in schemas['objects']:
+            if self._seen_id(id(obj)):
+                continue
+            self._convert_object_schema(obj)
+        for type_list, schema_yamls in schemas.items():
+            for schema in schema_yamls:
+                if self._seen_id(id(schema)):
+                    continue
+                if type_list == 'objects':
+                    continue
+                if type_list == 'arrays' or type_list == 'arrays':
+                    NameConverter._convert_language_default_name(schema)
+                    NameConverter._convert_language_default_name(schema['elementType'])
+                elif type_list == 'constants':
+                    NameConverter._convert_language_default_name(schema)
+                    NameConverter._convert_language_default_name(schema['value'])
+                    NameConverter._convert_language_default_name(schema['valueType'])
+                elif type_list == 'sealedChoices' or type_list == 'choices':
+                    self._convert_enum_schema(schema)
+                else:
+                    NameConverter._convert_language_default_name(schema)
+
+    def _convert_enum_schema(self, schema):
+        schema['language']['python'] = dict(schema['language']['default'])
+        schema_name = schema['language']['default']['name']
+        schema_python_name = schema['language']['python']['name']
+        schema_python_name = NameConverter._to_camel_case(schema_name)
```

Comment:
you're defining twice "schema_python_name"

Reply:
thanks!